hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -1553,6 +1553,18 @@
   </description>
 </property>
 
+<property>
```

Comment:
Entry in core-default.xml is avoidable. This can be documented via javadoc. (Every new entry in core-default adds to the Configuration object memory overhead)

Reply:
never really considered that. I've been more driven by what people can use with the XML tools to generate a list of configuration options.

Reply:
I'm removing this, and add the description of this config to the docs.